config/partner_details.py

```diff
@@ -63,8 +63,8 @@
      "desc":lazy_gettext("HireGo is building the world's first decentralized peer-to-peer car rental platform using Origin Protocol.")
     },
     {
-    "name":"pinprotocol",
-    "url":"https://pinprotocol.io/",
+    "name":"layerprotocol",
+    "url":"https://layerprotocol.com/",
      "desc":lazy_gettext("The decentralized reputation, incentive, and payments protocol for the global sharing economy, built on the blockchain.")
```

Comment:
@coleman415 should this be "Layer is the..." or "Layer Protocol is the..."? Their logo doesn't have their name in it.